.gitignore

```diff
@@ -1,11 +1,19 @@
+# Python development, test, and build
```

Comment:
Mostly I just added some comments to the .gitignore

The substantive change is the the **.vscode** directory containing configuration for Visual Studio Code is now ignored.